src/main/webapp/scripts/nav-bar.js

```diff
@@ -0,0 +1,17 @@
+
+function loadNav(){
+    $("#nav-placeholder").load("../nav.html", changeBar);
```

Comment:
I like how you used the html file as a template.  I will use this approach for the comments section. :)